docs/control-structures.rst

```diff
@@ -562,16 +563,10 @@ of an exception instead of "bubbling up".
     if the account called is non-existent, as part of the design
     of the EVM. Account existence must be checked prior to calling if needed.
 
-Exceptions in external calls can be caught with the ``try``/``catch`` statement.
-
-Exceptions can contain data that is passed back to the caller.
-This data consists of a 4-byte selector and subsequent :ref:`ABI-encoded<abi>` data.
-The selector is computed in the same way as a function selector, i.e.,
-the first four bytes of the keccak256-hash of a function
-signature - in this case an error signature.
-
-Currently, Solidity supports two error signatures: ``Error(string)``
-and ``Panic(uint256)``. The first ("error") is used for "regular" error conditions
+Exceptions can contain error data that is passed back to the caller
+in the form of :ref:`error instances <errors>`.
+The built-in errors ``Error(string)`` and ``Panic(uint256)`` are
+used by special functions, as explained below. The first ("error") is used for "regular" error conditions
```

Comment:
Shouldn't it be `The first ("Error(string)") is used..` here? Also the next line should be `while the second `Panic(uint256)` is used..` I think it makes it more clear then.

Reply:
I'd say just
```markdown
`Error` is used for "regular" error conditions while `Panic` is used for...
```

Reply:
This section evolved so much over time, I think it is due for a full rewrite at some point...